proto/streamlit/proto/Selectbox.proto

```diff
@@ -39,6 +39,7 @@ message Selectbox {
   LabelVisibilityMessage label_visibility = 10;
   string placeholder = 11;
   optional bool accept_new_options = 12;
+  optional string filter_mode = 15;
 
-  // Next: 15
```

Comment:
This somehow said 15 even though the values only go to 13. Is that related to the width value from flex layout? Or is it a bug?